template/setup.cfg.jinja

```diff
@@ -0,0 +1,98 @@
+[metadata]
```

Comment:
I notice when I create a plugin I have duplicate information in both `setup.cfg` and `pyproject.toml`, it would be good if we could get rid of `setup.cfg` entirely and just use `pyproject.toml`? I tried deleting the `setup.cfg` file from the resulting plugin directory and then building the plugin distribution and it worked just fine

Reply:
Will do, thanks